### ISSUE TITLE
Private Conversation

### ISSUE THREAD

**Issue by MsaadDL** [FossifyOrg/Messages#48]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Mark Conversation as Private

- Hide the Conversation From Conversation list
- Add a private Conversation list
- Open the private list by Passcode or Biometric 
- Hide the new message Details on the notification and Lock screen

### Why do you want this feature?

Have more privacy on specific Conversations

**Comment by inson1** [comment]
messages are now saved in your device database, where would Fossify saved this conversations?

**Comment by MsaadDL** [comment]
Yes its saved in the phone , What i mean on the App interface to have a list for normal conversations and a list for private conversations .

**Comment by naveensingh** [comment]
@MsaadDL Why not just lock the whole app using`Password protect the whole application` option? It's simpler and does the job.

As @inson1 mentioned, if you have another messaging app on your device, the private conversation is only a few taps away.

**Comment by naveensingh** [comment]
We would have to store conversations in-app and ignore the official Android database for them to be somewhat private.

**Comment by Zokii0** [comment]
> We would have to store conversations in-app and ignore the official Android database for them to be somewhat private.

Does this apply for, for example, making separated list of hidden conversations? I was looking if someone asked if it's possible to hide certain conversations which we mute through custom notifications, so we don't see those conversations on main page with other conversations, but to rather look at them later in separated hidden list.

**Comment by naveensingh** [comment]
Closing as it would just be a misnomer and not truly private. 

**Password protect the whole application** option is already there for the general case.
